### DISCUSSION TITLE
Breakpoints past md: don't work

### DISCUSSION THREAD

**Discussion by Pensive1** [tailwindlabs/tailwindcss#11747] [Help]
Hi team,

I've configured my tailwind config with the following screen sizes:
```
theme: {
    screens: {
      sm: "320px",
      md: "744px",
      xl: "1280",
      "2xl": "1440px",
    },
    ...
}
```

I'm trying to apply the following grids to these breakpoints:

- sm (mobile): 4 columns
- md (tablet): 8 columns
- xl (1280 desktop):12 columns
- 2xl (1440+ desktop):  12 columns.

Apart from grids, I've noticed other styles for xl and 2xl aren't applying to my body tag. However, my fonts are scaling for all breakpoints. I'm tempted to hard-code the grid rules in the global CSS file.

Reference: [https://github.com/Pensive1/portfolio/blob/190d70f44039c3a748fe1cc7dca22f77e79113f7/app/(site)/layout.tsx#L29](code snippet)

Text (to copy)
```
<div className="grid grid-cols-4 gap-x-4 md:grid-cols-8 xl:grid-cols-12 xl:gap-x-6">
...
</div>
```
I'm not sure why it's not working. Even chat-gpt gave me the same code.

**Comment by wongjn** [comment]
Your reference link leads to a 404 – it could be that the repository is private.

Either way, your `xl` `screen` configuration needs a unit to be a valid `min-width` media query. Judging from your other configuration values, perhaps `px`:

```diff
 screens: {
   sm: "320px",
   md: "744px",
-  xl: "1280",
+  xl: "1280px",
   "2xl": "1440px",
},
```
[Working example on Tailwind Play](https://play.tailwindcss.com/W4iq4FSEKG?file=config)

**Comment by Pensive1** [comment]
Perfect! It's fixed now.

After inspecting it in Chrome dev tools, I noticed my grid was going off the screen. I gave the container a width and it was sorted.

Thanks for the fast reply! 🙂
